.github/workflows/self-push.yml

```diff
@@ -96,11 +96,6 @@ jobs:
           echo "env.CI_BRANCH = ${{ env.CI_BRANCH }}"
           echo "env.CI_SHA = ${{ env.CI_SHA }}"
 
-      - name: Checkout transformers
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 2
-
```

Comment:
This created another `transformers` directory (we already have it when using our docker image )